mamar-wasm-bridge/src/lib.rs

```diff
@@ -68,6 +71,7 @@ pub fn bgm_encode(bgm: &JsValue) -> JsValue {
 #[wasm_bindgen]
 pub fn ron_encode(bgm: &JsValue) -> JsValue {
     let bgm: Bgm = from_js(bgm);
+    let bgm = bgm_ron::Bgm::from(bgm);
```

Comment:
I don't want this line anyway, but it would be better written (imo) as `let bgm: bgm_rom::Bgm = bgm.into();`